packages/typescript/src/typescript.ts

```diff
@@ -106,8 +107,11 @@ export function typescriptΔ(configFilePath: string, extraCompilerOptions: ts.Co
       ...extraCompilerOptions
     };
     config.compilerOptions = fullCompilerOptions;
-    config.files = filePaths;
-    delete config.include;
+
+    if (!config.compilerOptions.incremental) {
+      config.files = filePaths;
+      delete config.include;
+    }
```

Comment:
From my tests last night, a single-file compile with or without incremental set is still significantly faster than incremental even if no files have changed. I'm just filtering out that weird error and calling it good 😅